### PR TITLE
Make tooltip larger for latest build

### PR DESCRIPTION
**Pull request by YodaLightsabr**
Fixes the text expanding out of the tooltip as seen in this image:

image

### CODE REVIEW

File: app/assets/stylesheets/components/_tooltips.scss [hunk 1]
```diff
@@ -30,6 +30,10 @@
       transition: 0.125s all ease-in-out;
       width: max-content;
       z-index: 1;
+
+      &.tooltipped--lg {
```
Comment (garyhtou) [inline app/assets/stylesheets/components/_tooltips.scss]: What are your thoughts on increasing the `max-width` on line 21?

Comment (YodaLightsabr) [reply app/assets/stylesheets/components/_tooltips.scss]: That should work; I can check if it breaks other things.

**Review by garyhtou — APPROVED** [review]

**Comment by cjdenio** [comment]
This does make certain tooltips very large

Screenshot 2022-12-24 at 1 45 01 PM

**Comment by garyhtou** [comment]
> This does make certain tooltips very large

oh yeah- that's something I didn't consider. @YodaLightsabr maybe your [original commit](https://github.com/hackclub/bank/pull/3217/commits/ca02f9addfcf652ee3f7ed5bbae4a176683b72a1) might make more sense. I'll leave this up to you to decide!

**Comment by maxwofford** [comment]
I've reverted back to @YodaLightsabr's original commit and am now merging because this works locally!

**Pull request merged**